### DISCUSSION TITLE
Error while installing: Unsupported file *.deb

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#1136] [Help]
(EDIT due to command line error)

Trying to install grandorgue_0.3.1.2334+155.1_amd64.deb using apt install on a Ubuntu 20.04.4 LTS system I get error messages:

Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'grandorgue' instead of '/home/jens/Downloads/grandorgue_0.3.1.2334+155.1_amd64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 grandorgue : Depends: grandorgue-resources but it is not installable
              Recommends: grandorgue-demo but it is not installable
              Recommends: wx3.0-i18n but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

Any explanation from the guru´s? I hvae followed https://github.com/GrandOrgue/grandorgue/blob/master/INSTALL.md

(I know this is an old GO version, but I have some testing to do ...)

**Comment by oleg68** [comment]
The old versions of grandorgue were destributed in three seperate packages: grandorgue, grandorgue-resources and grandorgue-demo, so you have to download and install all of them with one apt-get command. See #741 for details

**Reply by jason-barba** [reply]
Thanks. Now I know why. But I cannot find grandorgue-resources nor grandorgue-demo packages, neither here on github nor on sourceforge. Most greatful for a hint.

//Jens

**Reply by jason-barba** [reply]
I got the result I needed from this site: https://software.opensuse.org/download/package?project=home:e9925248:grandorgue&package=grandorgue

Still interested in where the deb packeges are to found, though.

//Jens

**Reply by oleg68** [reply]
https://software.opensuse.org/download/package?project=home:e9925248:grandorgue&package=grandorgue-resources
https://software.opensuse.org/download/package?project=home:e9925248:grandorgue&package=grandorgue-demo

**Reply by jason-barba** [reply]
Thank you, Oleg!
